crates/x_ai/src/x_ai.rs

```diff
@@ -72,6 +80,8 @@ impl Model {
             Self::Grok3MiniFast => "Grok 3 Mini Fast",
             Self::Grok4 => "Grok 4",
             Self::GrokCodeFast1 => "Grok Code Fast 1",
+            Self::Grok4Fast => "Grok 4 Fast",
+            Self::Grok4FastThinking => "Grok 4 Fast Thinking",
```

Comment:
I think it might be better to align the display name to the official docs:

https://docs.x.ai/docs/models/grok-4-fast-reasoning
https://docs.x.ai/docs/models/grok-4-fast-non-reasoning

```suggestion
            Self::Grok4FastReasoning => "Grok 4 Fast",
            Self::Grok4FastNonReasoning => "Grok 4 Fast (Non-Reasoning)",
```

Would you agree?